modules/ROOT/pages/creating-and-managing-a-cluster-manually.adoc

```diff
@@ -153,6 +155,19 @@ The JMX monitoring option is disabled by default. To enable it, add the `+mule.c
 
 Please note that enabling JMX might cause some performance overheads given that the underlying structure adds listeners to get the statistics for each individual node.
 
+[[table-name-transformers]]
+===== Table Name Transformers
+
+The `mule.cluster.jdbcstoretableNametransformerstrategy` property enables you to define a custom transformer to modify table names.
+
+For example, if you set the following property in the `mule-cluster.properties` file:
+[source]
+----
+mule.cluster.jdbcstoretableNametransformerstrategy=com.mulesoft.mule.runtime.module.cluster.api.persistence.query.MD5TableNameTransformerStrategy
+----
+Mule hashes table names using MD5 and a prefix to identify that these are Mule tables. Hashing table names guarantees that the length constraint is honored.
+
+Optionally, you can create a custom transformer strategy by implementing the `com.mulesoft.mule.runtime.module.cluster.api.persistence.query.TableNameTransformerStrategy` interface.
```

Comment:
I think we mean "Alternatively," rather than "Optionally,"
Do we want to state that it is easier to set the property in the file rather than in the interface, or does it matter?